QSB/TransformSync/QuaternionHelper.cs

```diff
@@ -1,33 +1,33 @@
 ﻿using UnityEngine;
 
-namespace QSB
+namespace QSB.TransformSync
 {
-    static class Helpers
+    public static class QuaternionHelper
     {
         // Stolen from here: https://gist.github.com/maxattack/4c7b4de00f5c1b95a33b
         public static Quaternion QuaternionSmoothDamp(Quaternion rot, Quaternion target, ref Quaternion deriv, float time)
```

Comment:
If the file is now called Quaternion helper, we can rename this method to just `SmoothDamp`